app/Model/Category.php

```diff
@@ -11,6 +11,22 @@ class Category extends Model
     use Sluggable, SoftDeletes;
 
     /**
+    * Return value of parameter
```

Comment:
"Return value of parameter" nó ko đúng, nên comment:
    /**
     * The table associated with the model.
     *
     * @var string
     */